lesson-02/hw2/example-02.js

```diff
@@ -0,0 +1,30 @@
+const calculateEngravingPrice = function(message, pricePerWord) {
+  const MSGARR = message.split(' ');
+  const res = MSGARR.length * pricePerWord;
```

Comment:
Тут можна одразу повертати значення, не створюючи додаткової змінної 💭 
```suggestion
  return MSGARR.length * pricePerWord;
```